### PR TITLE
Hema

### PR DESCRIPTION
**Pull request by hemavmane**
vaccine data

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,54 @@
-import logo from './logo.svg';
-import './App.css';
+import { Dna } from "react-loader-spinner";
+import { createContextApi } from "./Component/ContextApiProvider1/ContextApiProvider1";
```
Comment (Souvikdeb2612) [inline src/App.js]: This is an internal import

File: src/Component/FdaApprrove/Fda.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      .get( `https://vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com/api/vaccines/get-fda-approved-vaccines`,
+        {
+             headers: {
+            'X-RapidAPI-Key': '8c0c5645f4msh17b4e89de7b874ap10eb20jsn3e95f4dad049',
+            'X-RapidAPI-Host': 'vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com'
+          },
+        }
+      )
+      .then(res => {
+        console.log(res.data);
```
Comment (Souvikdeb2612) [inline src/Component/FdaApprrove/Fda.jsx]: remove/comment console statements before pushing

File: src/Component/FdaApprrove/Fda.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <div className="heading_vaccine1">
+          <h3 className="all_h31 Co1">Company ▼</h3>
+          <h3 className="all_h31 pha1">Phase ▼</h3>
+          <h3 className="all_h131 Cat1">Category ▼</h3>
+          <h3 className="all_h131 Cat1">Description▼</h3>
+        </div>
+        <table className="All_vaccine_sidebar1">
+          <div className="vaccine_data_body">
+            <tbody>
+              {fda.map(i => {
```
Comment (Souvikdeb2612) [inline src/Component/FdaApprrove/Fda.jsx]: destructure i here

File: src/Component/Phase1/Phase.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+          headers: {
+            "X-RapidAPI-Key":
+              "8c0c5645f4msh17b4e89de7b874ap10eb20jsn3e95f4dad049",
+            "X-RapidAPI-Host":
+              "vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com",
+          },
+        }
+      )
+      .then(res => {
+        console.log(res.data);
```
Comment (Souvikdeb2612) [inline src/Component/Phase1/Phase.jsx]: same as above

File: src/Component/Phase1/Phase.jsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        <div className="heading_vaccine1">
+          <h3 className="all_h31 Co1">Company ▼</h3>
+          <h3 className="all_h31 pha1">Phase ▼</h3>
+          <h3 className="all_h131 Cat1">Category ▼</h3>
+          <h3 className="all_h131 Cat1">Description▼</h3>
+        </div>
+        <table className="All_vaccine_sidebar1">
+          <div className="vaccine_data_body">
+            <tbody>
+              {phase1.map(i => {
```
Comment (Souvikdeb2612) [inline src/Component/Phase1/Phase.jsx]: same as above

File: src/Component/VaccineData/Vaccine.jsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+          headers: {
+            "X-RapidAPI-Key":
+              "8c0c5645f4msh17b4e89de7b874ap10eb20jsn3e95f4dad049",
+            "X-RapidAPI-Host":
+              "vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com",
+          },
+        }
+      )
+      .then(res => {
+        console.log(res.data);
```
Comment (Souvikdeb2612) [inline src/Component/VaccineData/Vaccine.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Component/FdaApprrove/Fda.jsx (hunk 2 above)
Comment (Souvikdeb2612) [reply src/Component/FdaApprrove/Fda.jsx]: It is still not fixed

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Review by Souvikdeb2612 — APPROVED** [review]

File: src/Component/FdaApprrove/Fda.jsx (hunk 2 above)
Comment (hemavmane) [reply src/Component/FdaApprrove/Fda.jsx]: removed

**Pull request merged**